.travis.yml

```diff
@@ -5,23 +7,30 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - nightly
   - hhvm
 
 matrix:
+  fast_finish: true
+  include:
+    - php: 5.6
+      env: SYMFONY_VERSION=2.4.* PHPUNIT_VERSION=~3.7
+    - php: 5.6
+      env: SYMFONY_VERSION=2.0.* PHPUNIT_VERSION=~4.0
+    - php: 5.6
+      env: SYMFONY_VERSION=2.1.* PHPUNIT_VERSION=~4.0
+    - php: 5.6
+      env: SYMFONY_VERSION=2.2.* PHPUNIT_VERSION=~4.0
+    - php: 5.6
+      env: SYMFONY_VERSION=2.3.* PHPUNIT_VERSION=~4.0
+    - php: 5.6
+      env: SYMFONY_VERSION=2.4.* PHPUNIT_VERSION=~4.0
```

Comment:
This change massively decreases the number of jobs per build (one build now has 12 jobs instead of 30).